dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/impl/validation/resources/ValidationMessages.nlsprops

```diff
@@ -32,6 +32,14 @@ infoTermsOfServiceInvalidURL=The Info object must contain a valid URL. The \"{0}
 invalidUrl=The \"{0}\" object must contain a valid URL. The \"{1}\" value specified for the URL is not valid.
 invalidUri=The \"{0}\" object must contain a valid URI. The \"{1}\" value specified for the URI is not valid.
 
+headerExampleOrExamples="example" and "examples" specified in header parameter \"{0}\" are mutually exclusive."
+headerSchemaAndContent=Header parameter \"{0}\" MUST contain either a schema property, or a content property, but not both.
+headerSchemaOrContent=Header parameter \"{0}\" MUST contain either a schema property or a content property.
+headerContentMap=The 'content' map within header parameter \"{0}\" must only contain one entry.
```

Comment:
It's good to avoid using single quote in messages (ID reviewers don't like that). Instead use the double quotes (escaped).
'content'  --> \"content\"